ansible/roles/skywalking/vars/main.yml

```diff
@@ -15,3 +15,10 @@
 
 ---
 skywalking_version: "9.5.0"
+sw_server_port: "8080"
+sw_oap_server_port: 12800
+
+env_vars:
+  SW_SERVER_PORT: "{{ sw_ui_server_port }}"
+  SW_OAP_ADDRESS: "{{ hostvars[groups['skywalking-oap'][0]].ansible_host }}:{{ sw_oap_server_port }}"
```

Comment:
Some hints:

- We can configure multiple addresses here when in cluster mode, e.g, http://1.2.3.4:12800,http://1.2.3.5:12800
- We need `http://` prefix for each of the OAP addresses, e.g., http://1.2.3.4:12800

Reply:
Is it ok to address the cluster mode in the next PR? It might require changes in the way we handle ansible inventory.